osu.Game.Rulesets.Osu.Tests/Resources/old-skin/skin.ini

```diff
@@ -1,2 +1,6 @@
 [General]
-Version: 1.0
\ No newline at end of file
+Version: 1.0
+
+[Fonts]
+HitCircleOverlap: 3
+ScoreOverlap: 3
```

Comment:
where do these values come from? should they not be defaults instead?

Reply:
got them from an osu! 2007 skin I downloaded, didn't wanna make them incorrectly spaced out, doesn't matter anyway?

Reply:
it's weird, is all. i'd be expecting defaults from an old default skin. not a huge deal i suppose.